students/jemielity_kamil/lesson_02_flow_control/knight_move.py

```diff
@@ -0,0 +1,19 @@
+
+starting_column = int(input('Start column: '))
+starting_row = int(input('Start row: '))
+
+ending_column = int(input('Ending column: '))
+ending_row = int(input('Ending row: '))
+
+if abs(starting_column - ending_column) == 2:
+    if ending_row == starting_row + 1 or starting_row - 1:
+        print('YES')
+    else:
+        print('NO')
+elif abs(starting_row - ending_row) == 2:
```

Comment:
My first impression is that 'elif abs(starting_row - ending_row) == 2:' should be merged with 'if abs(starting_column - ending_column) == 2:'

I noticed that this code doesn't pass snakify test. Are you using Snakify tests to verify your solution? If not, please do